bundles/org.openhab.binding.cul.max/README.md

```diff
@@ -0,0 +1,204 @@
+| pair_mode       | Switch             | Turns on / off the pair_mode on the MAX!CUL. It will switch to off after 1 minute | maxcul_bridge |
+| listen_mode     | Switch             | Turns on / off the listen_mode. In listen mode all messages are just logged. | maxcul_bridge |
+
+## Status
+
+The binding is currently in beta and it is recommended that you only use it expecting there to be bugs and issues. It is has enough features to be useful as a heating system, though lacks some of the finer features. This page will be updated as things progress.
+
+## Tutorial
+
+There was a tutorial using a Raspberry Pi with OH1 available at [technpol](https://technpol.wordpress.com/2016/04/09/configuration-of-maxcul-and-cul-dongle/), which addresses some of the configuration issues.
+
+## Features
+
+The binding currently offers the following features:
+
+* Listen mode - this allows you to listen in on MAX! network activity from a MAX!Cube for example. A trace will be output in debug mode that decodes implemented messages
+* Pairing - can pair devices with OpenHAB by triggering Pair Mode using a Switch item
+* Wall Thermostat
+* Can send set point temperature
+* Can receive set point temperature
+* Can receive measured temperature
+* Can receive battery status
+* Can receive operating mode
+* Can factory reset device
+* Can be configured to display current temperature or current setpoint (_likely 1.8.0+_)
+* Radiator Thermostat Valve
+* Can send set point temperature
+* Can receive set point temperature
+* Can receive measured temperature
+* Can receive valve position
+* Can receive battery status
+* Can receive operating mode
+* Can factory reset device
+* Push Button
+* Can receive either AUTO or ECO depending on button press (translated to ON/OFF)
+* Can factory reset device
+* Association
+* It is possible to link devices together so that they communicate directly with each other, for example a wall thermostat and a radiator valve.
+* TX Credit Monitoring
+
+## Limitations
+
+Aside from understanding what the binding does do which is documented here there are some key things to be aware of that may limit what you hope to achieve.
+
+1. Radiator thermostat data is updated quite sporadically. Items such as set point temperature, measured temperature, valve position, battery status and operating mode are only sent when the state of the valve changes - i.e. valve moves or the dial used to manually set a temperature. If you want measured temperature it is much better to use a wall thermostat.
+1. The binding has no concept of 'auto' mode: It currently has no ability to retrieve from any source and subsequently send a schedule to devices. This may change in the future, which would allow basic operation should OpenHAB fail for some reason.

```

Comment:
```suggestion
1. The binding has no concept of 'auto' mode: It currently has no ability to retrieve from any source and subsequently send a schedule to devices. This may change in the future, which would allow basic operation should openHAB fail for some reason.

```

Reply:
done.